package.json

```diff
@@ -27,14 +27,23 @@
     "daemon": "^1.1.0",
     "express": "^4.13.4",
     "juttle": "^0.4.0",
-    "juttle-service": "^0.1.0",
+    "juttle-service": "juttle/juttle-service",
     "juttle-viewer": "0.0.1",
```

Comment:
Maybe change this to juttle/juttle-viewer for now as well so we can pull in fixes. We'll lock down to specific versions before release.